Gemfile

```diff
@@ -42,6 +47,10 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'dotenv-rails'
```

Comment:
Line 50 and 51 should move between line 13 and 15 as they are windows specific gems